### PR TITLE
Api requirement change

### PR DESCRIPTION
**Pull request by lejoe**

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -3,20 +3,24 @@ import difflet from 'difflet';
...
-			style: true,
-			documentation: true
-		},
-		'elements/grid': {
-			style: true,
-			documentation: true
+const expected = {
+	elements: {
+		colors: {
+			path: 'src/elements/colors/',
```
Comment (lucalanca) [inline test.js]: @lejoe is path redudant here given that `src` was already passed (as config) and `elements/colors/` is the "path" on the returned object?

File: test.js [hunk 2]
```diff
@@ -3,20 +3,24 @@ import difflet from 'difflet';
...
+			style: 'styles.scss',
```
Comment (lucalanca) [inline test.js]: what happens if there is more than one `*.scss` file? Should this be an array?

Comment (lejoe) [inline test.js]: nah I would let this as a convention for now.
we only include the default styles.scss.

If you want more files you can but use _file1.scss _file2.scss and included them in styles

File: test.js (hunk 1 above)
Comment (lejoe) [inline test.js]: My logic is that those as well as the folders passed in the config should be relative to the base path. './fixtures/simple' in this example

**Pull request merged**